src/components/BoardSizeForm.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import Error from './Error';
+
+const BoardSizeForm: React.SFC<BoardFormProps> = props => {
+  const { handleChange, handleSubmit, error } = props;
+  return (
+    <React.Fragment>
+      <p>What size should the board be?</p>
+      <form id='size-form' onSubmit={handleSubmit}>
+        Board Size:
+        <input onChange={handleChange} type='text' name='size' />
+        <input id='size-submit' type='submit' value='submit' />
+      </form>
+      {error && <Error />}
```

Comment:
If `error` is a boolean, convention would usually call it `isError` or `hasError` or something less ambiguous; I'm not sure if typescript obviates that convention or not, but I would still err toward "less ambiguous," types or no.